### ISSUE TITLE
im shield badges broken for repos created on new docker hub

### ISSUE THREAD

**Issue by jsmedmar** [docker/hub-feedback#1733]
See https://github.com/badges/shields/issues/2533. It seems the API is not returning an updated value for `is_automated`, not sure what's going on in the case for `repo not found`.

**Comment by rfgamaral** [comment]
Having the same issue, another example here: https://github.com/rfgamaral/docker-gphotos-uploader/blob/master/README.md

Could we get this looked at?

**Comment by ldez** [comment]
I created a new shield.io badge https://github.com/badges/shields/pull/3139

**Comment by jsmedmar** [comment]
Can confirm this was solved by @ldez in #1400. Fix your badges with:

## Automated

```diff
-https://img.shields.io/docker/automated/{user}/{repo}.svg
+https://img.shields.io/docker/cloud/automated/{user}/{repo}.svg
```

## Status 
```diff
-https://img.shields.io/docker/build/{user}/{repo}.svg
+https://img.shields.io/docker/cloud/build/{user}/{repo}.svg
```

It worked for me 👍
